recipes/mss/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.2.0" %}
+
+package:
+    name: mss
+    version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    skip: True  # [py3k]
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+    build:
+        - python
+        # mss import pyproj from basemap and that does not exist in 1.0.8
+        - basemap 1.0.7
+        - chameleon
+        - execnet
+        - lxml
+        - netcdf4
```

Comment:
@jakirkham yes the netcdf 1.2.2 version has only the curl issue for me. There are SSLv2 symbols missing which were in older versions like curl==7.26. The curl issue only occures with an ssl connection. 

The netcdf 1.2.4 version gives other issues and those break both kinds of my server setup (http and https not working)

 ```
File "/home/mss/config/mss_wms_settings.py", line 56, in <module>
import netCDF4
File "/home/mss/miniconda2/lib/python2.7/site-packages/netCDF4/__init__.py", line 3, in <module>
from ._netCDF4 import *
libmfhdf.so.0: cannot open shared object file: No such file or directory
```